### ISSUE TITLE
Add graph flow view to dependencies in maestro++

### ISSUE THREAD

**Issue by jkotalik** [dotnet/arcade#2438]
It would be nice to have a graph view in maestro++, showing the dependencies and direct dependents for a repository. Here is a mock up:

![image](https://user-images.githubusercontent.com/8302101/55584662-989f9d00-56d9-11e9-80e7-3efcad6cad06.png)

**Comment by jcagme** [comment]
Hey @jkotas yes, this would be great, also, we have the ability of enabling that on each official build. See ["Generating Graph Files"](https://github.com/dotnet/arcade/blob/master/Documentation/AzureDevOps/GeneratingGraphFiles.md) for reference.

Currently CoreSDK and Arcade builds have this feature enabled. Ping me and I can show you the generated files.

**Comment by markwilkie** [comment]
For example, to access the graph picture in core-sdk, go to:
1)	Dnceng internal project
2)	Open the ‘DotNet Core SDK (Official)’ build pipeline
3)	Choose the build you’re interested in 
4)	Click on ‘Artifacts’ on the top right and choose ‘GraphFiles’

E.g. https://dev.azure.com/dnceng/internal/_build/results?buildId=141003

**Comment by jkotalik** [comment]
That's cool! For starters, I will start by trying to add that to aspnetcore repos. 

However, I do believe that https://maestro-prod.westus2.cloudapp.azure.com/3/https:%2F%2Fgithub.com%2Faspnet%2FAspNetCore/6860 is a significantly better view. The thing we care about with this view is "How many days has it been since we last updated our downstream dependencies" With the current graph view, the data is there but it is hard to parse/interpret. Also, it isn't readily available (something we could put on a dashboard in a team room, for example).

**Comment by markwilkie** [comment]
@alexperovich - Could you point @jkotalik to the maestro website and help get him permissions please?

**Comment by alexperovich** [comment]
@jkotalik is a member of the team. He should be able to access https://maestro-prod.westus2.cloudapp.azure.com.

**Comment by markwilkie** [comment]
Looks like there's nothing more to do here
